### PR TITLE
LLMEval performance

### PR DESCRIPTION
**Pull request by davidkoski**
- notes about running under Xcode
- reminder about models having different performance
- don't update the display for every token

### CODE REVIEW

File: Applications/LLMEval/ContentView.swift [hunk 1]
```diff
@@ -198,7 +203,7 @@ class LLMEvaluator {
             let prompt = modelConfiguration.prepare(prompt: prompt)
             let promptTokens = MLXArray(tokenizer.encode(text: prompt))
 
-            let initTime = Date()
+            var initTime = Date()
```
Comment (awni) [inline Applications/LLMEval/ContentView.swift:206]: I guess you don't really need to initialize it here?

File: Applications/LLMEval/README.md [hunk 2]
```diff
@@ -50,3 +50,9 @@ Building in Release / optimizations will remove a lot of tail calls in the C++
 layer.  These lead to the stack overflows.
 
 See discussion here: https://github.com/ml-explore/mlx-swift-examples/issues/3
+
+### Performance
+
+Different models have difference performance characteristics.  For example gemma may outperform phi2 in terms of tokens / second.
```
Comment (awni) [inline Applications/LLMEval/README.md:56]:
```suggestion
Different models have difference performance characteristics. For example Gemma 2B may outperform Phi-2 in terms of tokens / second.
```

File: Applications/LLMEval/README.md [hunk 3]
```diff
@@ -50,3 +50,9 @@ Building in Release / optimizations will remove a lot of tail calls in the C++
...
+
+You may also find that running outside the debugger is also a performance boost.  You can do this in xcode by pressing cmd-opt-r and uncheck "Debug Executable".
```
Comment (awni) [inline Applications/LLMEval/README.md:58]:
```suggestion
You may also find that running outside the debugger boosts performance.  You can do this in Xcode by pressing cmd-opt-r and unchecking "Debug Executable".
```

**Review by awni — APPROVED** [review]
Minor comments, looks great! Thanks for debugging the perf issues! 🚀

File: Applications/LLMEval/ContentView.swift (hunk 1 above)
Comment (davidkoski) [reply Applications/LLMEval/ContentView.swift:206]: It is used on the line below to show the time spent preparing, so this is still needed.

**Pull request merged**